src/views/AbstractView.ts

```diff
@@ -98,6 +98,12 @@ abstract class AbstractView {
           continue;
         }
 
+        if (entity.entity_category === "config" &&
+          (Helper.strategyOptions.domains[this.#domain ?? ""].hide_config_entities ||
```

Comment:
`this.#domain ?? ""` is not the way to avoid TypeScript errors.
It should be validated against `undefined` instead, since "" is a valid object key.

Reply:
It is valid but practically it is unlikely to be present in the configuration. 

In the edge case where an entity is missing domain having a `"":` section in the configuration, while not ideal, is not the worst idea either. Alternatively we can switch it with something like `NO_DOMAIN` or `EMPTY`.

Reply:
In practice `this.#domain` will never be undefined, but TypeScript cannot know that at compile-time.
Although it's unlikely a user will ever use `""`, `NO_DOMAIN`, `EMPTY` or anything else non-expected as an identifier in their configuration, it's still a possibility on which we don't want to act.

You can ass `this.#domain` as an condition to the `if` statement:
```typescript
        if (
          entity.entity_category === "config" &&
          this.#domain &&
          (Helper.strategyOptions.domains[this.#domain].hide_config_entities ||
            Helper.strategyOptions.domains.default.hide_config_entities)
        ) {
          continue;
        }
```

Reply:
We could use `_`. That one, we know is meaningful.

... or I can use this check. Your call.

Reply:
Either way is fine with me when using `_` as the identifier for all domains.

However, make sure `Helper.strategyOptions.domains["_"]` is always defined.
File `configurationDefaults.ts` is probably the best place to declare it.